setup.cfg

```diff
@@ -4,16 +4,20 @@ force_grid_wrap = 0
 include_trailing_comma = True
 known_first_party = transformers
 known_third_party =
+    absl
     fairseq
     fastprogress
     git
+    h5py
     MeCab
     nltk
+    numpy
     packaging
     PIL
     psutil
     seqeval
     sklearn
+    tensorflow
     tensorboardX
     tensorflow_datasets
     torchtext
```

Comment:
For consistency, you could add torch as well, which is just as optional as tensorflow is.

Reply:
indeed, thanks